test/NuGet.Clients.FuncTests/NuGet.CommandLine.FuncTest/Commands/RestoreCommandTests.cs

```diff
@@ -16,6 +17,9 @@ namespace NuGet.CommandLine.FuncTest.Commands
 {
     public class RestoreCommandTests
```

Comment:
Why not add these tests?
```
Restore_LegacyPackageReference_WithCustomNamedNuGetLockFile
Restore_LegacyPackageReference_WithNuGetLockFileLockedMode
Restore_LegacyPackagesConfig_ForceEvaluateNuGetLockFile
```

Reply:
packages.config projects don't do anything with the `ForceEvaluate` argument, it's a PackageReference only feature.

I added the other two tests.